### PR TITLE
Update translation for the DATA-FILES.md changes.

### PR DESCRIPTION
**Pull request by tsanie**
## PR Checklist
**Please check if your PR fulfills the following requirements:**

- [ ] The commit message follows [our guidelines](https://github.com/theme-next/hexo-theme-next/blob/master/.github/CONTRIBUTING.md).
- [ ] Tests for the changes have been added (for bug fixes / features).
   - [ ] Muse | Mist have been tested.
   - [ ] Pisces | Gemini have been tested.
- [x] Docs have been added / updated (for bug fixes / features).

## PR Type
**What kind of change does this PR introduce?**  

- [ ] Bugfix.
- [ ] Feature.
- [ ] Code style update (formatting, local variables).
- [ ] Refactoring (no functional changes, no api changes).
- [ ] Build related changes.
- [ ] CI related changes.
- [x] Documentation content changes.
- [ ] Other... Please describe:

## What is the new behavior?
Update translation for the DATA-FILES.md changes   (de3eea692b34af82f452c365a75ba4abf7974cb8).

## Does this PR introduce a breaking change?
- [ ] Yes.
- [x] No.

### CODE REVIEW

File: docs/cn/DATA-FILES.md [hunk 1]
```diff
@@ -10,14 +10,52 @@
 
 如果你仍然希望使用 Hexo 2.x，你依旧可以按老的方式进行配置。NexT 仍然兼容 Hexo 2.x（但可能会出现错误）。
 
-## 带来的好处
+<h2 align="center">选择 1：Hexo 方式</h2>
 
-使用这一特性，你现在可以将你的全部配置置于同一位置（`source/_data/next.yml`），并且不需要修改 `next/_config.yml`。如果在新的 release 中出现了任何新的选项，那么你只需要从 `next/_config.yml` 中将他们复制到 `source/_data/next.yml` 中并设置它们的值为你想要的选项。
+使用这一方式，你的全部配置都将置于 hexo 主要配置文件中（`hexo/_config.yml`），并且不需要修改 `next/_config.yml`，或者创建什么其他的文件。但是所有的主题选项必须放置在 `theme_config` 后。
```
Comment (wafer-li) [inline docs/cn/DATA-FILES.md]: These words in the original `DATA-FILES.md` not translated:

```
But you must preserve double spaces indents within theme_config option.
```
Especially the **double spaces** not translated.

Comment (tsanie) [reply docs/cn/DATA-FILES.md]: Actually yeah, I didn't thought it might be make some misunderstanding.

**Comment by tsanie** [comment]
Yes, now the translation is ok.

**Comment by ivan-nginx** [comment]
Sure?

**Comment by tsanie** [comment]
Yeah. 😃 

There is an issue in the first commit, about the translation of 

> But you must **preserve double spaces indents** within theme_config option.

It has been fixed in the after (1b714923f2e48a0974ce838b549edbae9dade2b5).

**Comment by wafer-li** [comment]
After a double check, I cannot find any obvious bug on this.

I am sure it's ready to be merged

**Pull request merged**